src/node/og/og.c

```diff
@@ -0,0 +1,251 @@
+#include <netlibc/log.h>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wsign-conversion"
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wconversion"
+#define STB_TRUETYPE_IMPLEMENTATION
+#include "stb_truetype.h"
+#pragma GCC diagnostic pop
+#pragma GCC diagnostic pop
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wsign-conversion"
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wconversion"
+#define STB_IMAGE_WRITE_IMPLEMENTATION
+#include "stb_image_write.h"
+#pragma GCC diagnostic pop
+#pragma GCC diagnostic pop
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wsign-conversion"
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wconversion"
+#define STB_IMAGE_IMPLEMENTATION
+#include "stb_image.h"
+#pragma GCC diagnostic pop
+#pragma GCC diagnostic pop
+
+unsigned char *ttf_buffer = NULL;
+unsigned char *logo_data = NULL;
+int logo_width = 0;
+int logo_height = 0;
+int logo_channels = 0;
+
+#define TTF_BUFFER_SIZE 1 << 20
+#define IMG_WIDTH 1132
+#define IMG_HEIGHT 566
+#define ICON_SIZE 250
+
+#define MARGIN_LEFT 415
+#define PADDING_HORIZONTAL 94
+#define PADDING_VERTICAL 138
+
+#define LINE_HEIGHT 1.225f
+
+#define BACKGROUND_COLOR_R 134
+#define BACKGROUND_COLOR_G 62
+#define BACKGROUND_COLOR_B 0
+
+#define LOGO_FILEPATH "./node/explorer/static/img/icon/icon-250x250.png"
+#define FONT_FILEPATH "./node/explorer/static/font/Roboto-Regular.ttf"
```

Comment:
The paths should not be hardcoded.
Use utility functions to obtain the explorer path, then append the relative paths.